Rakefile

```diff
@@ -75,3 +75,123 @@ EOM
   end
 end
 
+namespace :acceptance do
+  require 'puppet_litmus/rake_tasks'
+  require_relative './spec/support/acceptance/helpers'
+  include TargetHelpers
+
+  desc 'Provisions the VMs. This is currently just the master'
+  task :provision_vms do
+    if File.exist?('inventory.yaml')
+      # Check if a master VM's already been setup
+      begin
+        uri = master.uri
+        puts("A master VM at '#{uri}' has already been set up")
+        next
+      rescue TargetNotFoundError
+        # Pass-thru, this means that we haven't set up the master VM
+      end
+    end
+
+    provision_list = ENV['PROVISION_LIST'] || 'acceptance'
+    Rake::Task['litmus:provision_list'].invoke(provision_list)
+  end
+
+  # TODO: This should be refactored to use the https://github.com/puppetlabs/puppetlabs-peadm
+  # module for PE setup
+  desc 'Sets up PE on the master'
+  task :setup_pe do
+    master.bolt_run_script('spec/support/acceptance/install_pe.sh')
+  end
+
+  desc 'Sets up the Splunk instance'
+  task :setup_splunk_instance do
+    puts("Starting the Splunk instance at the master (#{master.uri})")
+    master.bolt_upload_file('./spec/support/acceptance/splunk', '/tmp/splunk')
+    master.bolt_run_script('spec/support/acceptance/start_splunk_instance.sh')
+    # HEC token is hard coded because it will always be the same in the splunk container
+    instance, hec_token = "#{master.uri}:8088", 'abcd1234'
+
+    # Update the inventory file
+    puts('Updating the inventory.yaml file with the Splunk HEC credentials')
+    inventory_hash = LitmusHelpers.inventory_hash_from_inventory_file
+    splunk_group = inventory_hash['groups'].find { |g| g['name'] =~ %r{splunk} }
+    unless splunk_group
+      splunk_group = { 'name' => 'splunk_nodes' }
+      inventory_hash['groups'].push(splunk_group)
+    end
+    splunk_group['targets'] = [{
+      'uri' => instance,
+      'config' => {
+        'transport' => 'remote',
+        'remote' => {
+          'hec_token' => hec_token,
+        }
+      },
+      'vars' => {
+        'roles' => ['splunk_instance'],
+      }
+    }]
+    write_to_inventory_file(inventory_hash, 'inventory.yaml')
+  end
+
+  desc 'Installs the module on the master'
+  task :install_module do
+    Rake::Task['litmus:install_module'].invoke(master.uri)
+  end
+
+  # splunk_hec.yaml must be deleted from the master if this is run more than once.
```

Comment:
I'd go ahead and do this b/c I see no good reason to have the user do this.. Maybe `master.run_shell('rm -rf <splunk_hec.yaml>')`.

Reply:
adding this to the `install_module` task, and removing the `configure_module` task.